fastapi/demo/fastapi_endpoint_demo.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 ACSONE SA/NV
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL). -->
+<odoo>
+  <record model="fastapi.endpoint" id="fastapi_endpoint_demo">
+    <field name="name">Fastapi Demo Endpoint</field>
+    <field
+            name="description"
+        ><![CDATA[
+# A Dummy FastApi Demo
+
+## HOWTO
+
+This demo endpoint has been created by inhering from "fastapi.endpoint", registering
+a new app into the app selection field and implementing the `_get_fastapi_routers`
+methods.
+
+```python
+from typing import List
+
+from fastapi import APIRouter
+
+from odoo import fields, models
+
+
+class FastapiEndpoint(models.Model):
+
+    _inherit = "fastapi.endpoint"
+
+    app: str = fields.Selection(
+        selection_add=[("demo", "Demo Endpoint")], ondelete={"demo": "cascade"}
+    )
+
+    def _get_fastapi_routers(self) -> List[APIRouter]:
```

Comment:
```suggestion
    def get_fastapi_routers(self) -> List[APIRouter]:
```
IMHO, the (documentation)[https://github.com/acsone/rest-framework/blob/16.0-fastapi/fastapi/README.rst?plain=1#L222] is correct and this code must be changed